tests/test_workers/test_pkg_managers/test_general_js.py

```diff
@@ -14,6 +14,19 @@
 from cachito.workers.paths import RequestBundleDir
 
 
+def test_get_git_prefixes():
```

Comment:
Optional: I know this sounds a little obsessive, but the whole Python module is alphabetized so it'd be nice to keep this consistent. :stuck_out_tongue: